content/docs/user-guide/visualization/environments/terrain/terrain-developer-guide/future-work.md

```diff
@@ -0,0 +1,55 @@
+---
+linktitle: Future Work
+title: Future Work
+description: Possible future improvements to the terrain system.
+weight: 500
+toc: true
+---
+
+There are a number of improvements that could potentially be made to the terrain system for anyone interested in taking the system further.
+
+## Large-scale improvements
+
+This category of improvements requires architecting new solutions to provide extensive new capabilities.
+
+### 3D geometry support
+
+Instead of a heightfield-based approach, the APIs and renderer could be extended to use a full 3D approach, whether it's through voxels, displacement maps, mesh ingestion, raytracing, or some other technology. This would enable the system to support caves, arches, cliff geometry, and other vertical or overlapping terrain features that simply can't be supported through a pure heightfield approach.
```

Comment:
For my edification, can you explain the use of "will" instead of "would" here? Since this is a hypothetical scenario that hasn't happened yet, isn't "would" the correct choice?

i.e. "If I extend it, it would support caves" vs "If I extend it, it will support caves".

Reply:
Oh yes, sorry for the miss. Given the hypothetical or conditional statement, "would" is more accurate.